frontend/src/utilities/js-messages.ts

```diff
@@ -0,0 +1,256 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable camelcase */
+/* eslint-disable max-classes-per-file */
+
+import { Transform, Type } from "class-transformer";
+
+export class JsMessage {
+	static responseMarker = true;
```

Comment:
What does `responseMarker` do? Please comment its purpose, thanks.

Reply:
added the comment, "The marker provides a way to check if an object is a sub-class constructor for a jsMessage."